extras/userdoc/md/documentation/scheduling-and-simulation-flow.md

```diff
@@ -77,9 +80,17 @@ wide without need leads to decreased performance due to more update calls and
 communication cycles (small *d<sub>min</sub>*), or increased memory consumption
 of NEST (large *d<sub>max</sub>*).
 
-Related topics
---------------
+## Spike generation and precision
+
+A neuron fires a spike when the membrane potential at the end of an update interval of
+the simulation resolution is above threshold. For most models, the membrane potential
```

Comment:
I'd reorder the sentence a little: "A neuron fires a spike when the membrane potential is above threshold at the end of an update interval (i.e., a multiple of the simulation resolution)."